tools/kochdocs.nim

```diff
@@ -101,9 +104,9 @@ doc/lib.rst
 doc/manual.rst
 doc/manual_experimental.rst
 doc/destructors.rst
-doc/tut1.rst
-doc/tut2.rst
-doc/tut3.rst
+doc/tutorials\tut1.rst
```

Comment:
The backslashes need to be forward slashes .. `doc/tutorials\tut1.rst` -> `doc/tutorials/tut1.rst`, and the same elsewhere in this PR.

From https://dev.azure.com/nim-lang/Nim/_build/results?buildId=2228&view=logs&jobId=ba7bbfa4-f55c-5c34-6d52-1b6b4edd3f37&j=ba7bbfa4-f55c-5c34-6d52-1b6b4edd3f37&t=9859ff42-7c6c-5e13-a955-d92d36755b5f:

> Error: unhandled exception: cannot open: doc/tutorialstut1.rst [IOError]

Reply:
@kristianmandrup Pinging again as you probably missed this comment.